src/PageBanner/README.md

```diff
@@ -13,6 +13,14 @@ notes: |
 
 A ``Page Banner`` displays an important, succinct message, and provides actions for users to address (or dismiss the banner). It requires a user action to be dismissed.
 
+## Available Variants
+``dark``
+``light``
+``warning``
+``accentA``
+``accentB``
```

Comment:
Hmm, I think it'd make sense to instead of adding this new "Available Variants" section, add an example of using `accentB` in the existing "Variants" section below.

Alternatively, there is a `ExamplePropsForm` component that could be used here as well, which will allow you to configure a yellow box listing out all supported variants where the user can switch between them. See [this example](https://paragon-openedx.netlify.app/components/annotation/#referring-to-other-elements) or the below screenshot.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/2828721/196197562-255f651b-178a-4196-b089-b39824ed4917.png">

Reply:
added prop box

Reply:
@mehaknasir

<img width="956" alt="image" src="https://user-images.githubusercontent.com/2828721/196711703-0cec052c-2216-4681-bbca-46c96ec1c9be.png">


If we're using the prop box here, I think it'd make sense to include the rendered `PageBanner` controlled by the prop box below the prop box instead of above. There should also be some vertical spacing between the example and the prop box.

If we're keeping the existing pre-rendered variants (current below the prop box), we should add the `accentB` variant there, too. That said, I think if we're using the prop box we should get rid of the other pre-existing variant examples since they are now largely redundant with the example driven by the prop box.

Reply:
Thanks, Adam, I was thinking the same but was unsure if I should remove the current examples. I have removed those please review it again and let me know if that's okay or need some further improvements,